src/Areas/Storage/Commands/DataLake/Directory/DirectoryCreateCommand.cs

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using AzureMcp.Areas.Storage.Models;
+using AzureMcp.Areas.Storage.Options.DataLake.Directory;
+using AzureMcp.Areas.Storage.Services;
+using AzureMcp.Commands.Storage;
+using AzureMcp.Services.Telemetry;
+using Microsoft.Extensions.Logging;
+
+namespace AzureMcp.Areas.Storage.Commands.DataLake.Directory;
+
+public sealed class DirectoryCreateCommand(ILogger<DirectoryCreateCommand> logger) : BaseFileSystemCommand<DirectoryCreateOptions>
```

Comment:
It should not inherit from BaseFileSystemCommand because it does not require the parameter `FileSystem`